shared/src/constants/chainParameters.ts

```diff
@@ -0,0 +1,24 @@
+interface AddEthereumChainParameter {
+  chainId: string; // A 0x-prefixed hexadecimal string
+  chainName: string;
+  nativeCurrency: {
+    name: string;
+    symbol: string; // 2-6 characters long
+    decimals: 18;
+  };
+  rpcUrls: string[];
+  blockExplorerUrls?: string[];
+  iconUrls?: string[]; // Currently ignored.
```

Comment:
Why add it if we aren't using it?

Reply:
Just copy pasting the entire type into the definition